build/Linux+BSD/portable/copy-libs

```diff
@@ -79,8 +79,9 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
   #copyLib libQt5WebChannel.so.5  # Not present on Debian, and apparently not needed
-  copyLib libQt5WebKit.so.5
-  copyLib libQt5WebKitWidgets.so.5
+  copyLib libQt5WebEngine.so.5
+  copyLib libQt5WebEngineCore.so.5
+  copyLib libQt5WebEngineWidgets.so.5
```

Comment:
Your guess is as good as mine. @lasconic may have a better idea of what's needed.
Also see changes to Windows dependencies in [commit 4875862](https://github.com/musescore/MuseScore/commit/4875862aa19abdf204609b8e679bba3cbc785e0c).